docs-2.0/3.ngql-guide/1.nGQL-overview/1.overview.md

```diff
@@ -67,3 +67,95 @@ CREATE {TAG | EDGE} {<tag_name> | <edge_type>}(<property_name> <data_type>
 ```ngql
 nebula> CREATE TAG player(name string, age int);
 ```
+
+## 关于openCypher兼容性
+
+## 原生 nGQL 和 openCypher 的关系
+
+原生 nGQL 是由 Nebula Graph 自行创造和实现的图查询语言。OpenCypher 是由 openCypher Implementers Group 组织所开源和维护的图查询语言，
+最新版本为 openCypher 9。由于 nGQL 语言部分兼容了 openCypher，这个部分在本文中称为 OpenCypher 兼容语句。
```

Comment:
The last "OpenCypher" should not be capitalized.

Reply:
let's ignore the minor just for now.
And proofread later.